src/Symfony/Component/PropertyAccess/PropertyAccess.php

```diff
@@ -19,39 +21,30 @@
 final class PropertyAccess
 {
     /**
-     * Creates a property accessor with the default configuration.
+     * Creates a property accessor.
+     *
+     * For dealing with several property accessor configured differently, use
+     * the createPropertyAccessorBuilder() method instead.
      *
-     * @param bool $throwExceptionOnInvalidIndex
-     * @param bool $magicCall
+     * @param bool                        $throwExceptionOnInvalidIndex
+     * @param bool                        $magicCall
+     * @param CacheItemPoolInterface|null $cacheItemPool
      *
      * @return PropertyAccessor The new property accessor
      */
-    public static function createPropertyAccessor($throwExceptionOnInvalidIndex = false, $magicCall = false)
+    public static function createPropertyAccessor($magicCall = false, $throwExceptionOnInvalidIndex = false, CacheItemPoolInterface $cacheItemPool = null)
```

Comment:
It was already before, just more restrictive due to the missing arguments.
Also I understand your point of view about the usefulness of this, but most of time there is no need of the builder itself because these arguments are defined once only when getting the PropertyAccessor instance. IMO the builder should be used to deal with differently configured property accessor, enabling features via the builder methods rather than re-pass all the PropertyAccessor constructor args each time you want to change one of its properties..